Hummingbird.docc/Tutorials/MongoKitten/MongoKitten-1-Setup.tutorial

```diff
@@ -0,0 +1,137 @@
+@Tutorial(time: 15) {
+    @Intro(title: "Create a Hummingbird + Fluent application.") {
+        Create a simple web application using the Hummingbird template.
+        @Image(source: "mongokitten.png", alt: "MongoKitten logo")
```

Comment:
I think you should use hummingbird logo here and use mongokitten logo where you are doing mongokitten related stuff